Assets/Scripts/TapeInteract.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.InputSystem;
+
+/// <summary>
+/// This script should be added on the PlayerControler Object.
+/// Usess raycasts to see whats in front of the player, to be interacted with.
+/// Shows tooltip if Player can interact with object
+/// 
+/// Requires: The tape object tag to be set to "Tape"
+/// </summary>
+
+
+public class TapeInteract : MonoBehaviour {
```

Comment:
Follow code convention (5. Open brackets start on the next line.)

```suggestion
public class TapeInteract : MonoBehaviour 
{
```